src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -76,45 +82,50 @@ class HearingsDetailsMappingTest extends HearingsMappingBase {
     private VenueService venueService;
 
     public static final String PROCESSING_VENUE_1 = "test_place";
+    private static final String PHONE_NUMBER = "07483871426";
 
-    private String epimsId1;
-    private String epimsId2;
-    private String epimsId3;
-    private String epimsId4;
-    private List<VenueDetails> epimsIdsList;
+    private static final String EPIMS_ID_1 = "112233";
+    private static final String EPIMS_ID_2 = "332211";
+    private static final String EPIMS_ID_3 = "123123";
+    private static final String EPIMS_ID_4 = "321321";
+    private static final List<VenueDetails> EPIMS_ID_LIST = Arrays.asList(
+        VenueDetails.builder().epimsId(EPIMS_ID_1).build(),
+        VenueDetails.builder().epimsId(EPIMS_ID_2).build(),
+        VenueDetails.builder().epimsId(EPIMS_ID_3).build(),
+        VenueDetails.builder().epimsId(EPIMS_ID_4).build());
+
+    private SscsCaseData caseData;
 
     @BeforeEach
     void setUp() {
-        epimsId1 = "112233";
-        epimsId2 = "332211";
-        epimsId3 = "123123";
-        epimsId4 = "321321";
+        caseData = SscsCaseData.builder()
+            .appeal(Appeal.builder().hearingOptions(HearingOptions.builder().build()).build())
+            .processingVenue(PROCESSING_VENUE_1)
+            .build();
 
-        epimsIdsList = Arrays.asList(
-            VenueDetails.builder().epimsId(epimsId1).build(),
-            VenueDetails.builder().epimsId(epimsId2).build(),
-            VenueDetails.builder().epimsId(epimsId3).build(),
-            VenueDetails.builder().epimsId(epimsId4).build());
+        caseData.getAppeal().setHearingSubtype(HearingSubtype.builder()
+            .wantsHearingTypeTelephone("yes")
+            .hearingTelephoneNumber(PHONE_NUMBER).build());
+        caseData.getAppeal().setHearingOptions(HearingOptions.builder().wantsToAttend("yes").build());
```

Comment:
Can you add these into the above builder for reability/to make future changes easier?
Such as:
```
        caseData = SscsCaseData.builder()
            .appeal(Appeal.builder()
                .hearingOptions(HearingOptions.builder()
                    .wantsToAttend("yes")
                    .build())
                .hearingSubtype(HearingSubtype.builder()
                    .wantsHearingTypeTelephone("yes")
                    .hearingTelephoneNumber(PHONE_NUMBER)
                    .build())
                .build())
            .processingVenue(PROCESSING_VENUE_1)
            .build();
```

Reply:
Done